docs/2gp_testing.rst

```diff
@@ -0,0 +1,123 @@
+Testing with Second-Generation Packaging
```

Comment:
This file needs to be added to the index in order to show up somewhere, right?

Reply:
Whoops. Adding it under Continuous Integration; discussion welcome.